docs/docs/guides/core-types/models.md

```diff
@@ -28,9 +29,10 @@ class YourModel(Model):
 ```
 
 You can call the model as usual with:
+
 ```python
 import weave
-weave.init('project-name')
+weave.init('intro-example')
```

Comment:
I changed every single project to `intro-example` so that everything goes to the same project when using the docs